### ISSUE TITLE
LARMOR : T-Jump : Operation via IBEX

### ISSUE THREAD

**Issue by GDH-ISIS** [ISISComputingGroup/IBEX#3175]
T-Jump cell is to be used on LARMOR. Control the equipment via IBEX/script.

Reference : 

https://github.com/ISISComputingGroup/ControlsWork/issues/403
https://github.com/ISISComputingGroup/IBEX/issues/2973

**Comment by GDH-ISIS** [comment]
The T-Jump cell hardware has been converted from using a 1 pin of a serial port to using a Keithley 2400 Source Meter.
https://github.com/ISISComputingGroup/IBEX/issues/2973

**Comment by GDH-ISIS** [comment]
Keithley 2400 ioc has been configured to talk to the device. Blocks were set up to try and control the device as desired. However, we failed at the last hurdle to write a value to the source output voltage. We mitigated the problem by writing the necessary Keithley string command to the general write PV. A python script to do the basic operation will now be written by the instrument scientist. We will be contacted if more help is needed.
